api/src/db/model/model.ts

```diff
@@ -47,13 +50,28 @@ export abstract class Model {
     };
 
     try {
-      const results = await this.pool.query(query);
-      return results.rows.map(r => transformKeysToCamelCase(r)).pop();
+      const results = await this.runQuery(query);
+      return results.pop();
     } catch (err) {
       const message = `Unable to fetch Profile with ID ${id}`;
       logger.error(`${message}, err = ${err.message}`);
 
       throw err;
     }
   }
+
+  async runQuery(query: Query): Promise<any[]> {
+    let client;
+
+    try {
+      client = await this.pool.connect();
+      const results = await client.query(query);
+
+      return results.rows.map(r => transformKeysToCamelCase(r));
```

Comment:
not a big deal at all but this could be called `return results.rows.map(transformKeysToCamelCase);` just the same as a fyi

Reply:
Good point. Will update.